crates/orderbook/openapi.yml

```diff
@@ -91,56 +92,58 @@ paths:
               schema:
                 $ref: "#/components/schemas/Order"
         404:
-          description: Order was not found
+          description: Order was not found.
     delete:
       deprecated: true
-      summary: Cancels order by marking it invalid with a timestamp.
+      summary: Cancel an order by marking it invalid with a timestamp.
       description: |
         The successful deletion might not prevent solvers from settling the order.
-        Authentication must be provided by providing an EIP-712 signature of an
-        "OrderCacellation(bytes orderUids)" message.
+        Authentication must be provided by providing an 
+        [EIP-712](https://eips.ethereum.org/EIPS/eip-712) signature of an
+        `OrderCancellation(bytes orderUids)` message.
```

Comment:
```suggestion
        `OrderCancellation(bytes orderUid)` message.
```

Just checked, and it looks like this was a typo in the original OpenAPI spec:

https://github.com/cowprotocol/services/blob/7c2da6d7a2b0e68c07610d277c415cbbb26a5085/crates/model/src/order.rs#L536-L539